source/slurm/overview.rst

```diff
@@ -332,6 +332,169 @@ allowed.
 
 Run the command ``sbatch-limits -h`` to see the other available options.
 
+.. _memory-usage:
+
+Determine and specify a memory limit for your jobs
+==================================================
+
+You can use the "report-mem" command in your job to get memory usage
```

Comment:
Should this be "the ``report-mem`` command" ?

Reply:
should have been ``report-mem`` and now, it is.